core/management/commands/queue_warm_cache.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+
+from chronam.core.models import Page
+from chronam.core import tasks
+
+_logger = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+    help = "Queue a celery request for each page from chronicling america to warm up the cache."
+
+    def handle(self, **options):
+        for page in Page.objects.all():
+	    try:
+                url = "https://chroniclingamerica.loc.gov%s" % page.url
+                _logger.info("queuing url %s"% url)
```

Comment:
As a general logging rule, don't force string interpolation: `logger.info('queuing URL: %s', url)` allows tools like Sentry to aggregate and avoids the overhead of all of that string processing if the current log level won't emit the message.

Reply:
Specific to this invocation, wouldn't it make more sense to simply display this output using e.g. `self.stdout.write` (https://docs.djangoproject.com/en/1.8/howto/custom-management-commands/) so it'd go to the user who runs the command rather than a logfile which may not be read or would be seen by someone else at a different time?